xenissuing-android-lib/src/test/java/com/example/xenissuing_android/XenCryptoUnitTest.kt

```diff
@@ -94,10 +94,10 @@ class XenCryptUnitTest{
     fun decryptTheCardData() {
         val xenditKey = xenditPublicKey
         val plain = "test"
-        val xenIssuing = XenIssuing(xenditKey)
-        val iv = xenIssuing.ivKeyGenerator()
-        val encryptedSecret = xenIssuing.encryption(plain, iv)
-        val decrypted = xenIssuing.decryptCardData(encryptedSecret, iv)
+        val session = SecureSession(xenditKey)
+        val iv = session.ivKeyGenerator()
+        val encryptedSecret = session.encryption(plain, iv)
```

Comment:
What is the `encryption` method? Do we need this?

Reply:
the `encryption` is the legacy method that exists in the XenCrypt, but I think for our card decrypt context we don't need it, but CMIIW @sammous